packages/sra-types/.npmignore

```diff
@@ -0,0 +1,6 @@
+.*
+yarn-error.log
+/scripts/
+/src/
+tsconfig.json
+/lib/monorepo_scripts/
```

Comment:
Remove this one, we don't have `monorepo_scripts` anymore in package `lib` dirs.